man/standard-options.xml

```diff
@@ -29,6 +31,16 @@
     </listitem>
   </varlistentry>
 
+  <varlistentry id='no-pager-255'>
```

Comment:
Hmm, I don't really like having "versioned" options in `standard-options`, especially when the option already exists, but just not hooked up with some tools before.

Reply:
Do you have a better idea? For this particular tool, the option appears in version 255. The fact that the same option appeared in 214 in some other tool is not really relevant. By using standard-options, we minimize the number of copies to at most one per version. Also, all those text are together in the same file, so if we decide to change the wording one day, we can do this easily. If we had something better than docbook xml, that could do composition or macros or something, we probably could avoid such duplication. But this is what we have right now.

Reply:
OK. This is convincing.